jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -111,12 +125,13 @@ def search(self):
         viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 5, 'fill': False}
         viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name='catalog_results')
 
+        return skycoord_table
```

Comment:
Was this added on purpose?

Reply:
yes, we don't set it as a traitlet anywhere or show it in the plugin UI, so I figured you might as well be able to access the table of results from the API method instead of having to dig into markers.  This technically won't be public yet now that I stripped out exposing it, so if you'd prefer, I can revert this change for now too.

Reply:
No, I think this is good to have. Just want to make sure. Thanks!